cmd/chainbridge/main.go

```diff
@@ -18,36 +18,36 @@ import (
 	"github.com/ChainSafe/ChainBridge/core"
 	msg "github.com/ChainSafe/ChainBridge/message"
 	log "github.com/ChainSafe/log15"
-	"github.com/urfave/cli"
+	"github.com/urfave/cli/v2"
 )
 
 var app = cli.NewApp()
 
 var cliFlags = []cli.Flag{
-	config.ConfigFileFlag,
-	config.VerbosityFlag,
-	config.KeystorePathFlag,
-	config.BlockstorePathFlag,
-	config.FreshStartFlag,
-	config.LatestBlockFlag,
+	&config.ConfigFileFlag,
+	&config.VerbosityFlag,
+	&config.KeystorePathFlag,
+	&config.BlockstorePathFlag,
+	&config.FreshStartFlag,
+	&config.LatestBlockFlag,
 }
 
 var generateFlags = []cli.Flag{
-	config.PasswordFlag,
-	config.Sr25519Flag,
-	config.Secp256k1Flag,
+	&config.PasswordFlag,
+	&config.Sr25519Flag,
+	&config.Secp256k1Flag,
 }
 
 var devFlags = []cli.Flag{
-	config.TestKeyFlag,
+	&config.TestKeyFlag,
 }
 
 var importFlags = []cli.Flag{
-	config.EthereumImportFlag,
-	config.PrivateKeyFlag,
-	config.Sr25519Flag,
-	config.Secp256k1Flag,
-	config.PasswordFlag,
+	&config.EthereumImportFlag,
+	&config.PrivateKeyFlag,
+	&config.Sr25519Flag,
+	&config.Secp256k1Flag,
+	&config.PasswordFlag,
```

Comment:
Small nitpick really, but I think it would make more sense to update `config/flags.go` to provide pointers to flags, instead of taking a pointer when we use them.